fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/ClientRepositoryWrapper.java

```diff
@@ -86,8 +87,21 @@ public Client getActiveClientInUserScope(Long clientId) {
     public Client getClientByAccountNumber(String accountNumber) {
         Client client = this.repository.getClientByAccountNumber(accountNumber);
         if (client == null) {
-            throw new ClientNotFoundException(accountNumber);
+            throw new ClientNotFoundException(accountNumber, "account.number");
         }
         return client;
     }
+
+    public Client getClientByClientId(final Long clientId, final String hierarchySearchString) {
```

Comment:
Wrong method name. It get clients by cliendId and hierarchy, no?

Reply:
Done, updated